webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -152,6 +154,26 @@ export const getCVFilterPackageGroups = (cvId, filterId, params) => get({
   url: api.getApiUrl('/package_groups'),
 });
 
+export const addCVFilterPackageRule = (filterId, params, handleSuccess) => post({
+  type: API_OPERATIONS.POST,
+  key: ADD_CONTENT_VIEW_PACKAGE_GROUP_FILTER_RULE,
+  url: api.getApiUrl(`/content_view_filters/${filterId}/rules`),
+  params,
+  handleSuccess,
+  successToast: () => __('Filter added'),
```

Comment:
Thoughts on having something like: "Package group added to filter" and similar for error toast.

Reply:
Yeah, that was an oversight, good catch.
Would Package group rule added to filter make sense?